lib/server.js

```diff
@@ -524,12 +536,21 @@ WalletService.prototype.joinWallet = function(opts, cb) {
   if (_.isEmpty(opts.name))
     return cb(new ClientError('Invalid copayer name'));
 
+  opts.derivationStrategy = opts.derivationStrategy || WalletUtils.DERIVATION_STRATEGIES.BIP45;
```

Comment:
The wallet could run on BIP44 but old clients will try to join using an m/45' derived xPrivKey. How could the server prevent this from happening? Do you suggest inferring the derivation strategy exclusively from the client version?